### PR TITLE
[T3A5][W10-A4] Wang Zexin

### PR DESCRIPTION
**Pull request by wangzexin**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+    public String addPrefix(String str){
+    	return LINE_DECORATOR + str;
+    }
+    
+    public String addPostfix(String str){
+    	return str + LINE_DECORATOR;
+    }
+
+	public String formatShowToUser(String m) {
```
Comment (hwkchia) [inline src/seedu/addressbook/ui/Formatter.java:73]: You should really start watching your indentations.

**Review by hwkchia — COMMENTED** [review]
A good attempt at applying SRP.

**Comment by hwkchia** [comment]
ACK by closing this PR.

File: src/seedu/addressbook/ui/Formatter.java (hunk 1 above)
Comment (wangzexin) [reply src/seedu/addressbook/ui/Formatter.java:73]: ok I'll work harder!

**Pull request closed without merging**